python/ml4ir/applications/classification/tests/test_classification_model.py

```diff
@@ -32,7 +33,8 @@ def test_csv_loss_metrics(self):
                         msg=f"Top5 Categorical_accuracy not in expected range."
                             f" Expected: {expected_acc} ± {tol}, Found: {self.metrics_dict[_metric]}")
 
-    def test_group_metrics_df(self):
+    @pytest.mark.skip(reason="Pending based on spike for EP evaluation")
+    def test_group_metrics_df(self):  # FIXME: Pending based on spike for EP evaluation
```

Comment:
Note sure if we want to have sfdc notions like "spike for EP evaluation".

Reply:
Corrected it